airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -379,6 +379,8 @@ definitions:
       datetime_format:
         description: The format of the datetime
         type: string
+      cursor_format:
```

Comment:
At the cost of being more verbose, I do think we should indicate that this is specifically around the date format of the cursor and we should indicate that in how this is named `cursor_datetime_format`